azure-pipelines-integration.yml

```diff
@@ -4,6 +4,7 @@ trigger:
 - master-vs-deps
 - release/*
 - features/*
+- feature/convertNameOf
```

Comment:
This shouldn't be merging down to master; can we fix the branch name and remove this?